src/Our.Umbraco.InnerContent/Web/UI/App_Plugins/InnerContent/js/innercontent.controllers.js

```diff
@@ -111,44 +167,46 @@ angular.module('umbraco.directives').directive('innerContentOverlay', [
             scope.config.editorModels = scope.config.editorModels || {};
             scope.currentItem = null;
 
-            var getContentType = function (guid) {
+            var getContentType = function (key) {
```

Comment:
Why is this changed to key? Is this now a cachekey of guid + blueprint id? If so, does that mean the inner equality check wouldn't be right?

Reply:
Umbraco keeps referring to the Guid as Key - was trying to be consistent.

With the cachekey - I'd hit an issue where if you first selected a ContentType, then selected a Blueprint (or vice-versa), the first one would be cached. So they needed different cachekeys.

There may be a better way? (Hence requesting a review 😉)

Reply:
I don't mind going with consistency then, so should be ok to update these methods as you have, but I might be tempted to expose a guid and key from the api endpoint for backwards compatibilities sake?